$NetBSD: patch-contrib_config.make-Darwin,v 1.1 2014/05/15 01:14:44 pho Exp $

Do not override XML_CATALOG_FILES on Darwin. That will only make
xmllint try to find a wrong catalog.

--- contrib/config.make-Darwin.orig	2018-01-31 01:13:24.000000000 +0000
+++ contrib/config.make-Darwin
@@ -1,8 +1,5 @@
 # Example configuration for Darwin / Mac OS X.
 
-# Work-around for Homebrew-based xmlto.
-export XML_CATALOG_FILES=/usr/local/etc/xml/catalog
-
 TIG_CFLAGS += -DHAVE_EXECINFO_H
 TIG_LDLIBS = -liconv
 
